docs/src/conversions_views.md

```diff
@@ -410,6 +410,29 @@ compare two (or more) grayscale images. See
 [Keeping track of location with unconventional indices](@ref) for more
 information.
 
+## StackedViews
+
+Sometimes it's helpful to combine several images into a single view for 
+further array like manipulation. 
+```jldoctest; setup = :(using Images)
+julia> img1 = ImageMeta(rand(2,2,2))
```

Comment:
How about something deterministic like `reshape(0.01:0.01:0.08, (2,2,2))`? That way it can pass the doctest. (Here and in the next line too.)